src/model/geom/track.py

```diff
@@ -50,3 +51,23 @@ def sense_closest(self, sensor: Sensor, active_segment: SegmentId) -> float:
                 if (d := sensor.check_distance(wall)) is not None:
                     return d
         raise RuntimeError("couldn't find any walls in any of the segments")
+
+    @cached_property
+    def boundaries(self) -> Tuple[Vec2, Vec2]:
+        """:return: Tuple[Vec2, Vec2] - upper left and lower right corners of rectangle enclosing track
```

Comment:
We have already used different docs style. (Neuroevolution, Individual, Neural Network). it would be nice to have consistent style of docs.